apps/backend/src/api/devices.py

```
@@ -153,3 +170,82 @@ async def get_device_summary_by_hostname(
+
+# System Monitoring Endpoints
+@router.get("/{hostname}/metrics")
+async def get_device_metrics_by_hostname(
+    hostname: str = Path(..., description="Device hostname"),
+    include_processes: bool = Query(False, description="Include process information"),
+    timeout: int = Query(60, description="SSH timeout in seconds"),
+    current_user=Depends(get_current_user)
+):
+    """Get comprehensive system performance metrics from a device"""
+    try:
+        return await get_system_info(hostname, include_processes, timeout)
```

Ruff (0.12.2)

Undefined name `get_system_info`

(F821)